src/audio/multimediaaudioplayer.cc

```diff
@@ -47,7 +46,6 @@ void MultimediaAudioPlayer::stop()
   if ( audioBuffer ) {
     audioBuffer->close();
     audioBuffer->setData( QByteArray() ); // Free memory.
```

Comment:
```suggestion
    audioBuffer->buffer().clear()
```

No idea what the comment is saying (by 🟪🐧), but Qbuffer::setData(bytearray) doesn't manage lifetime of external bytearray.

We use the Qbuffer::setData(char*,size) which manages an internal qbytearray.

It looks like freeing memory because temp obj got deleted when out of scope. And the audioBuffer previously uses the internal default bytearray which will be reset/cleared when assigning external bytearray.

Let's just do it the simple way that don't need to understand the above Qbuffer internals.